components/Smiley.tsx

```diff
@@ -19,12 +19,31 @@ const EMOJI_MAP: EmojiMap = {
 }
 
 const Smiley = ({ percentage }: Props) => {
+  const min = 0
+  const max = 10
   const matchingEmoji = Object.keys(EMOJI_MAP).find(
     (key) => parseInt(key) >= percentage * 100
   )
   if (!matchingEmoji) return null
 
-  return <span className={styles.large}>{EMOJI_MAP[matchingEmoji]}</span>
+  return (
+    <span
+      aria-label={`Satisfaction rating: ${
+        parseInt(matchingEmoji) / 10
+      }, ${min} is the worst rating, ${max} is the best rating`}
+      aria-valuemax={max}
+      aria-valuemin={min}
+      aria-valuenow={parseInt(matchingEmoji) / 10}
+      className={styles.large}
+      role="slider"
+      tabIndex={0}
+      title={`Satisfaction rating: ${
```

Comment:
See localization issue. Would prefer to avoid `title` as well. The rest of the attributes should allow the system to generate this type of string automatically?